edward/inferences/gan_inference.py

```diff
@@ -101,6 +101,12 @@ def initialize(self, optimizer=None, optimizer_d=None,
     self.train_d = optimizer_d.apply_gradients(grads_and_vars_d,
                                                global_step=global_step_d)
 
+    if self.logging:
+      summary_key = 'summaries_' + str(id(self))
+      tf.summary.scalar('loss_fake', self.loss_d, collections=[summary_key])
```

Comment:
Can you rename these to 'loss_discriminative' and 'loss_generative' respectively?